src/Components/SmartComponents/Modals/OsExposureReportModal/LifecycleSelect.js

```diff
@@ -25,25 +27,44 @@ const LifecycleSelect = ({
                     onClick={() => setIsSelectOpen(!isSelectOpen)}
                     isExpanded={isSelectOpen}
                     style={{ width: '261px' }}
-                    isDisabled={!minorVersionDetails[minorVersion].eus && !minorVersionDetails[minorVersion].aus}
+                    isDisabled={!fullVersionDetails.eus && !fullVersionDetails.aus}
                 >
                     {`RHEL ${minorVersion}: ${selectedValue}`}
                 </MenuToggle>
             )}
             onOpenChange={(isOpen) => setIsSelectOpen(!isOpen)}
             onSelect={(_, value) => {
+                let indexOfOsVersion;
+                let selectedOsVersionsCopy = [...selectedOsVersions];
+                selectedOsVersions.forEach((osVersion, index) => {
+                    if (Object.keys(osVersion)[0] === minorVersion) {
+                        indexOfOsVersion = index;
+                        return;
+                    }
+                });
+
+                if (value === 'Minor release') {
```

Comment:
one more nit-pick: I would consider adding some kind of enum `const SupportValue = { ... }` to not to repeat writing values every time and reduce the chance of bugs.